src/to-be-disabled.ts

```diff
@@ -23,7 +23,8 @@ function isElementDisabled(element: ReactTestInstance) {
   return (
```

Comment:
I would go for targeted check, we don't want this matcher to be triggerd by `editable` prop set on `View` or other views.

```suggestion
  if (getType(element) === "TextInput" && element?.props?.editable === false) {
    return true;
  }

  return (
```